app/util/address/index.js

```diff
@@ -225,7 +229,7 @@ export function isENS(name) {
  * @param {address} string
```

Comment:
```suggestion
 * @param {string} address The 42 character address composed of:
 * 2 ('0x': 2 char hex prefix) + 20 (last 20 bytes of public key) * 2 (as each byte is 2 chars in ascii)
```

<img width="516" alt="Screenshot 2023-08-30 at 13 31 17" src="https://github.com/MetaMask/metamask-mobile/assets/1881059/eea01eab-1aa8-4474-bf47-247e06434d5a">